### ISSUE TITLE
The `cmd + z´ not working on German keyboard

### ISSUE THREAD

**Issue by rebekkaorth** [handsontable/handsontable#10794]
### Describe the bug

When using e.g. a German keyboard layout, the shortcut `cmd + z` doesn't work. 
To undo the last action you have to instead use `cmd + y`. 

This is because on a German keyboard the Y-key is where the Z-key is on a English or American keyboard layout. 

### Handsontable version

14.1

### Your environment

Chrome

**Comment by PierreB40** [comment]
Since V14, the French layout (AZERTY) is not supported either, which is problematic.

This change is not mentioned in the release note, but I think it's an important point.

Hopefully, as many layouts as possible will be supported in the future. Because it's an important feature. Non-QWERTY users shouldn't be deprived of this feature.

**Comment by adrianszymanski89** [comment]
Hi @PierreB40 

Thank you for pointing that out. We will discuss this problem, and I'll let you know if there are plans for the fix in the near future.

**Comment by adrianszymanski89** [comment]
Hi @OlgaIOn 

It hasn't been put on our roadmap yet. We will update you on this issue as soon as it's solved.

**Comment by AMBudnik** [comment]
Hi @rebekkaorth @OlgaIOn @PierreB40 

We have a small update here. Some time ago, here https://github.com/handsontable/handsontable/pull/11027/files we fixed the generic issue and now we operate on - what you see is what you get - rule. This means it doesn't matter where the `Z` is on your keyboard - `CTRL/CMD + Z` undoes the action.

If you'd rather have `CTRL/CMD + Y` to undo (for those users who have a QWERTZ keyboard but rather use QWERTY placement), you'd need to [create an additional keyboard shortcut using our shortcut manager](https://handsontable.com/docs/javascript-data-grid/api/shortcut-context/#addshortcut). 

If you have any more questions or would like to learn how to create custom shortcuts, please contact us at support@handsontable.com.
